monitors/index.js

```diff
@@ -264,6 +266,7 @@ async function Poll(callback) {
     const medianizerPriceFeedConfig = JSON.parse(process.env.MEDIANIZER_PRICE_FEED_CONFIG);
 
     await run(
+      Logger,
```

Comment:
If called with `truffle exec ../monitors/index.js`, the default logger will be `Logger`.